users/forms.py

```diff
@@ -26,6 +28,62 @@ def clean(self):
         return cleaned_data
 
 
+class UpdateProfileModelForm(forms.ModelForm):
+    """ Form for updating the user's profile
+    """
+    class Meta:
+        # what table to use
```

Comment:
@alejbernlapis comment in here is not needed since this is in the django documentation. Just put comments on custom codes/modules